Fix course details scroll

heatsink opened the pull request:
Lookin' good.
<img width="282" alt="screen shot 2018-06-19 at 9 48 30 am" src="https://user-images.githubusercontent.com/9795754/41583852-fd4c6b10-73a5-11e8-832b-da14fa3689e5.png">
Added in time and professor to the title, added print for hours per week even though we don't get this from Nine. Perhaps it would still be valuable for exchange students to estimate all courses as a fixed value for hours per week, not sure what value we would choose though.

iOS happily compiles.

AndiAndreas commented:
As far as i know i added professor on bottom of description. Had no better idea at that moment. You might need to remove this. Maybe you already did but I cant looke into the code right now.

heatsink commented:
👍 I removed the professor from the bottom of the description.

At the time, the new alignment was intentional because the click looked a little when it was left aligned. 
(before)
![Uploading Screen Shot 2018-06-19 at 11.44.19 AM.png…]()

However, I reverted the change and everything is right aligned. The little click thing was inconsequential and it looks better when everything is left aligned.

I also noticed the font for ECTS was different, so both the ECTS and Hours per Week uses the same CiE footer style.
In this commit:
<img width="224" alt="screen shot 2018-06-19 at 11 49 50 am" src="https://user-images.githubusercontent.com/9795754/41590262-e5c6d56e-73b6-11e8-978e-5c41b7ca5866.png">

AndiAndreas commented:
Good now looks like before 

![screenshot_20180619-115204](https://user-images.githubusercontent.com/6030353/41590791-32127f94-73b8-11e8-960a-11a7b553e962.png)
